### PR TITLE
Pro-266: Surface DAO name on contribution table

### PR DESCRIPTION
**Pull request by NKoech123**
## Linear Ticket
https://linear.app/govrn/issue/PRO-266/surface-dao-name-on-the-contributions-feeds

## Description
-  Added a column(index=0) "DAO" on the contributions table
   Question/Doubt: Since this is a contribution table, does it look weird to have DAO at Column 0? 
                                Then looking at the sequence of the column headings from column 0 -> 1, reading "DAO" then "Name"
                                 may sound confusing to a user. I propose having the contribution name come first then followed by DAO.
                                 What do you think @jonathanprozzi @alexkeating @amrro . 

- Extracted guild name (DAO name) from contribution Data,  then passed the DAO name to its respective contribution

## Screencaptures
Screen Shot 2022-07-23 at 11 19 41 PM

### CODE REVIEW

**Review by amrro — CHANGES_REQUESTED** [review]
LGTM except minor changes and DAO's name column position.

File: apps/protocol-frontend/src/components/ContributionsTable.tsx [hunk 1]
```diff
@@ -59,12 +59,26 @@ const ContributionsTable = ({
...
+
       })),
     [contributionsData]
   );
-
+ 
   const columns = useMemo(
     () => [
+      {
+        Header: 'DAO',
```
[the remark this answers is not in the document]
Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/ContributionsTable.tsx:71]: I think DAO should likely go toward the end, possibly after _Attestations_. I don't think that the DAO is the first thing that the user is going to be looking for in this view. Other than that I'd say that this looks good!

File: apps/protocol-frontend/src/components/ContributionsTable.tsx [hunk 2]
```diff
@@ -59,12 +59,26 @@ const ContributionsTable = ({
         activityTypeId: contribution.activity_type.id,
         status: contribution.status.name,
         action: '',
+        guildName: !contribution.guilds.map((guildObj: any) => guildObj.guild.name)[0]? '---' : contribution.guilds.map((guildObj: any) => guildObj.guild.name)[0]
+
```
[the remark this answers is not in the document]
Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/ContributionsTable.tsx:63]: Agree with this! Definitely suggest simplifying

**Review by jonathanprozzi — CHANGES_REQUESTED** [review]
Looks good but let's move the DAO header to the end -- our original conception with the wireframes had DAO after Attestations I believe. We can also get some additional user feedback on this but I don't think it's the primary item that user's will want to see. Let's move to after Attestations and see how that feels.

File: apps/protocol-frontend/src/components/ContributionsTable.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (NKoech123) [reply apps/protocol-frontend/src/components/ContributionsTable.tsx]: Thanks @amrro and @jonathanprozzi  for the comments. I have addressed all your comments and pushed.
Actions on comments:
- Simplified line 62 (guildName: ....)
- removed comments on styling
- moved DAO column from "after contribution name" to "after Attestation". Here's how it's looking now:

Screen Shot 2022-07-25 at 11 29 39 AM

**Review by jonathanprozzi — APPROVED** [review]
Solid! Thanks for making those changes 🚀

**Pull request merged**